.github/PULL_REQUEST_TEMPLATE/pull-request--new-issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: 'Pull Request: New issue'
+about: Report and propose a fix for a (small) issue in Hazel
+title: Add a relevant PR title here.
+labels: ''
+assignees: ''
+
+---
+
+ **Describe the issue**
+A clear and concise description of what the issue is.
+A screenshot or copy of the error could be helpfull as well.
+
+ **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+ **Proposed fix**
+A short description of what fix this is. And how it fixed the issue you described.
+
+  **PR impact**
+List of related issues/PRs this will solve:
+
+ Impact                  | PR
+------------------------ | ------
+Other issues this solves | None or #number(s)
+Other PRs this solves    | None or #number(s)
+
+  **Additional context**
+Add any other context about the solution here. Did you test the sollution on all (relevant) platforms?
```

Comment:
sollution -> solution

Reply:
Fixed typo in https://github.com/TheCherno/Hazel/pull/80/commits/bda4f8f9e22f8887fb4b046c185349f9863368a4.